CreatePerson/function.json

```
@@ -6,7 +6,6 @@
       "direction": "in",
       "name": "req",
       "methods": [
-        "get",
         "post"
       ],
       "route": "test/"
```

```suggestion
      "route": "person/"
```
Give proper routes :)

---

just change the routes :) @liorPerezz